README.md

```diff
@@ -11,7 +11,7 @@ This basically is a Swift version of the iOS client found in this tutorial: ["Ho
 
 ### How to Try Out
 
- 1. Clone the repo: e.g. `git clone https://github.com/hideya/loopback-swift-simple-example`
+ 1. Clone the repo: e.g. `https://github.com/hm-91/loopback-swift-simple-example.git`
```

Comment:
Ah, putting the URL there was not good an idea as it has the dependency to the project owner's id.
How about just saying"1. Close the repo" ?